rbi/core/array.rbi

```diff
@@ -2460,6 +2505,23 @@ class Array < Object
   sig {returns(T::Array[Elem])}
   def to_ary(); end
 
+  # Returns the result of interpreting *ary* as an array of `[key, value]`
+  # pairs.
+  #
+  # ```ruby
+  # [[:foo, :bar], [1, 2]].to_h
+  #   # => {:foo => :bar, 1 => 2}
+  # ```
+  #
+  # If a block is given, the results of the block on each element of the array
+  # will be used as pairs.
+  #
+  # ```ruby
+  # ["foo", "bar"].to_h {|s| [s.ord, s]}
+  #   # => {102=>"foo", 98=>"bar"}
+  # ```
+  def to_h(&blk); end
```

Comment:
This is overriding the definition on `Enumerable`, and causing some errors in the stripe codebase. What do you think about just removing this one?